aspnetcore-webapi/HideEndpointInSwagger/HideEndpointInSwagger/Controllers/WeatherForecastController.cs

```diff
@@ -0,0 +1,64 @@
+using Microsoft.AspNetCore.Mvc;
+
+namespace HideEndpointInSwagger.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class WeatherForecastController : ControllerBase
+    {
+        private static readonly string[] Summaries = new[]
+        {
+          "Freezing", "Bracing", "Chilly", "Cool", "Mild", "Warm", "Balmy", "Hot", "Sweltering", "Scorching"
+        };
```

Comment:
Let's format this, by breaking it into multiple lines and use the array initializer: 
```suggestion
        private static readonly string[] Summaries = 
        {
            "Freezing",
            "Bracing",
            "Chilly",
            "Cool",
            "Mild",
            "Warm",
            "Balmy",
            "Hot",
            "Sweltering",
            "Scorching"
        };
```